Sources/MongoSwift/ChangeStreamOptions.swift

```diff
@@ -0,0 +1,91 @@
+/// Describes the modes for configuring the fullDocument field of a `ChangeStreamDocument`.
```

Comment:
similar to nit below, we may want to say "change stream document" rather than `ChangeStreamDocument`.